scripts/rpc.js

```diff
@@ -0,0 +1,12 @@
+function RPC(title) {
+    const client = require('discord-rich-presence')('963128360219869194');
```

Comment:
Will I need to create a new ID instead of 9631...194 ?

Reply:
No, its already created :)